### PR TITLE
Updated Back-end 

### PR DESCRIPTION
**Pull request by ysonggh**

### CODE REVIEW

File: TalkingToTheSpaceAngularNTierApp/DAL/DataContext/DatabaseContext.cs [hunk 1]
```diff
@@ -40,6 +40,7 @@ public OptionsBuild()
         public DbSet<User> Users { get; set; }
         public DbSet<Message> Messages { get; set; }
         public DbSet<Reply> Replies { get; set; }
+        public DbSet<Point> Points { get; set; }
```
Comment (ysonggh) [inline TalkingToTheSpaceAngularNTierApp/DAL/DataContext/DatabaseContext.cs:43]: The new table, called Point, was created in the database for further usage.

File: TalkingToTheSpaceAngularNTierApp/DAL/DataContext/DatabaseContext.cs [hunk 2]
```diff
@@ -58,10 +59,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<User>().Property(ap => ap.User_ID).UseIdentityColumn(1, 1).IsRequired().HasColumnName("user_id");
             //COLUMN SETTINGS
             modelBuilder.Entity<User>().Property(ap => ap.Username).IsRequired(true).IsUnicode().HasMaxLength(100).HasColumnName("username");
-            modelBuilder.Entity<User>().Property(ap => ap.User_Password).IsRequired(true).HasMaxLength(100).HasColumnName("user_password");
+            modelBuilder.Entity<User>().Property(ap => ap.User_Token).IsRequired(true).HasMaxLength(100).HasColumnName("user_token");
```
Comment (ysonggh) [inline TalkingToTheSpaceAngularNTierApp/DAL/DataContext/DatabaseContext.cs:62]: The database won't save password anymore. Instead, it saves user-token released from Auth0 Server.

File: TalkingToTheSpaceAngularNTierApp/DAL/DataContext/DatabaseContext.cs [hunk 3]
```diff
@@ -74,6 +75,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                    .WithOne(ap => ap.User)
                    .HasForeignKey(app => app.User_ID)
                    .OnDelete(DeleteBehavior.Restrict);
+            modelBuilder.Entity<User>()
+                .HasOne<Point>(app=>app.Point).WithOne(ap=>ap.User)
+                .HasForeignKey<Point>(app=>app.User_ID)
+                .OnDelete(DeleteBehavior.Restrict);
```
Comment (ysonggh) [inline TalkingToTheSpaceAngularNTierApp/DAL/DataContext/DatabaseContext.cs:81]: One-to-one relationship between User Entity and Point Entity.

File: TalkingToTheSpaceAngularNTierApp/DAL/obj/DAL.csproj.nuget.dgspec.json [hunk 4]
```diff
@@ -40,6 +40,10 @@
         "net5.0": {
           "targetAlias": "net5.0",
           "dependencies": {
+            "Microsoft.AspNetCore.Authentication.JwtBearer": {
+              "target": "Package",
+              "version": "[5.0.10, )"
+            },
```
Comment (ysonggh) [inline TalkingToTheSpaceAngularNTierApp/DAL/obj/DAL.csproj.nuget.dgspec.json:46]: Added Authentication Package to the backend.

**Review by ysonggh — COMMENTED** [review]
All conflicts of back-end code have been reviewed

**Review by slurpbig — APPROVED** [review]

**Review by slurpbig — APPROVED** [review]

**Pull request merged**